test/migration.test.js

```diff
@@ -246,6 +249,9 @@ describe('migrations', function() {
   });
 
   it('should autoupdate', function(done) {
+    if (mysql56) {
```

Comment:
I disagree about this part for 2 reasons:
1) it is not only about mysql version but there is the Platform (OS) factor as well. 
2) It is a temp solution, where we need to dig deeper to find a way to adjust date/decimal/number formats to the new mysql version